### DISCUSSION TITLE
local-ai - delete models

### DISCUSSION THREAD

**Discussion by BluePixel4k** [nextcloud/all-in-one#4545] [Questions] [answered]
Hi,

how do I delete downloaded models from local-ai?
Where are the models stored?
Removing the models from `models.yaml` isn't enough.

Thank you!

**Comment by szaimen** [comment, marked as answer]
Hi, the models are stored in this volume: https://github.com/nextcloud/all-in-one/blob/75fdb2848617836da2014244e08916766d4fcc90/community-containers/local-ai/local-ai.json#L17

**Reply by BluePixel4k** [reply]
Thx, but I can't find the exact location.
In `/var/lib/docker/volumes/` or inside the docker bash in `/var/www/html/custom_apps/` is no folder called models or similar.

**Reply by szaimen** [reply]
It should usually be /var/lib/docker/volumes/nextcloud_aio_localai_models

**Reply by BluePixel4k** [reply]
Oh, my mistake. I tried it with `sudo`, but I had to switch to the `root` user to see the volumes.
Thank you!
